packages/models-library/src/models_library/progress_bar.py

```diff
@@ -12,11 +12,11 @@ class ProgressStructuredMessage(BaseModel):
     description: IDStr
     current: float
     total: int
-    unit: str | None
+    unit: str | None = None
```

Comment:
why that change?

Reply:
Actually this is a change done by the `bump-pydantic` tool. I checkked the reason, it seems be there: https://github.com/pydantic/pydantic/issues/5481

Reply:
:tada: finally they differentiate between nullable and optional/required! I should extend a test a did long time ago about this topic. SEE https://github.com/ITISFoundation/osparc-simcore/pull/6333/files#r1761340739

Reply:
no sorry but I disagree @pcrespov @giancarloromeo please read the documentation.
To me that means now that it is an optional field with a default value of None. Which is and was not the intention originaly. it was a required field that allows to take a None value. Neverthelesss it is required. so let's discuss and if I'm right this need to be changed.